NadekoBot.Core/Modules/Utility/Utility.cs

```diff
@@ -203,12 +203,12 @@ public async Task Roles(IGuildUser target, int page = 1)
 
             const int rolesPerPage = 20;
 
-            if (page < 1 || page > 100)
```

Comment:
I don't think side effects in comparisons are ideal. This should just be a separate statement above. It looks like the function takes in ordinal page numbers but uses it as an index in the rest of the code.

Reply:
Sorry I meant to put this comment on change and not the old code

Reply:
i moved it outside the if statment.